swim/heal_partition_test.go

```diff
@@ -68,7 +68,7 @@ func TestPartitionHealWithFaulties(t *testing.T) {
 	assert.Len(t, targets, 1, "expected correct amount of targets")
 	assert.True(t, B.Contains(targets[0]), "expected target to be a node from the right partition")
 
-	waitForPartitionHeal(t, time.Second, A, B)
+	waitForPartitionHeal(t, 100, A, B)
```

Comment:
100 seconds is a lot; can we do anything to mock time to make sure it's guaranteed to be faster?